packages/trace-viewer/src/ui/watchMode.tsx

```diff
@@ -339,6 +337,7 @@ const TestList: React.FC<{
       fileNames.add(fileNameForTreeItem(treeItem)!);
     }
     sendMessageNoReply('watch', { fileNames: [...fileNames] });
+    innerSetWatchedTreeIds({ value: watchedTreeIds });
```

Comment:
Now when I click watch, it changes under cursor - it did not before. Covered with a test.